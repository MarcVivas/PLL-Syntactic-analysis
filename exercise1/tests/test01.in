c Exemple CNF DIMACS: mini-sudoku de 2 x 2
c CNF amb 4 variables i 6 clàusules

c Final de clàusula: 0


c This file should be correct







p cnf 4 6
c This file should be correct




1 2 0
-1 -2 0
c This file should be correct

c This file should be correct

3                   4
0



-1 -3 0 -3 -4 0


-2
-4
 0

