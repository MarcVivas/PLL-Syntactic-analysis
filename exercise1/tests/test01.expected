(p1 v p2) ^
(!p1 v !p2) ^
(p3 v p4) ^
(!p1 v !p3) ^
(!p3 v !p4) ^
(!p2 v !p4)
